hack/import-restrictions.yaml

```diff
@@ -143,16 +143,19 @@
   allowedImports:
   - k8s.io/apimachinery
   - k8s.io/apiserver
+  - k8s.io/component-base
```

Comment:
Can you also add component-base as a `baseImportPath` and then add apimachinery to the list of allowedImports?